README.md

```diff
@@ -205,6 +205,26 @@ Feel free to stop by our [Slack channel](https://empireslacking.herokuapp.com) (
 
 * The [SlithIR documentation](https://github.com/trailofbits/slither/wiki/SlithIR) describes the SlithIR intermediate representation.
 
+# FAQ
+
+* How do I exclude mocks or tests?
```

Comment:
Maybe we can link to the wiki here? (https://github.com/crytic/slither/wiki/Usage#path-filtering)